pkgs/tools/networking/distant/default.nix

```diff
@@ -0,0 +1,38 @@
+{
+  lib,
+  fetchFromGitHub,
+  rustPlatform,
+  perl,
+  pkg-config,
+  openssl,
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "distant";
+  version = "0.20.0";
+
+  src = fetchFromGitHub {
+    owner = "chipsenkbeil";
+    repo = pname;
+    rev = "v${version}";
+    hash = "sha256-DcnleJUAeYg3GSLZljC3gO9ihiFz04dzT/ddMnypr48=";
+  };
+
+  cargoHash = "sha256-7MNNdm4b9u5YNX04nBtKcrw+phUlpzIXo0tJVfcgb40=";
+  nativeBuildInputs = [
+    pkg-config
+    perl
+  ];
+  buildInputs = [ openssl ];
+  doCheck = false;
+
+  meta = {
+    description = "Distant - A fast, secure remote development tool";
+    homepage = "https://github.com/chipsenkbeil/distant";
+    license = lib.licenses.mit;
+    maintainers = [
+      "chipsenkbeil"
+      "westernwontons"
+    ];
```

Comment:
Maintainers are supposed to be handles defined in lib/maintainers.nix.

Check the header of that file and create a separate commit for each maintainer. The package commit should come after.

Reply:
i couldn't find the file you are talking about, also these are the maintainers of the project so logically they can be here

Reply:
The file is at `./maintainers/maintainer-list.nix`, but the references are available at `lib.maintainers`.

Reply:
so it should be like add maintainers to the maintainers/maintainer-list.nix
and add this line in default.nix?
```nix
maintainers = [ lib.maintainers.user ]
```
tell me if i'm corret cause i'm confused

Reply:
Yes, add a maintainer object at the correct location to the file and reference it as `lib.maintainters.chipseenkbeil` if you name the object that.

Reply:
thank you so much, big help
i just got done, tell me if i'm missing anything :)